.travis.yml

```diff
@@ -1,167 +1,100 @@
-#  Docker Hub user repository: user_name is the same as 
-#  the Travis Ci account owner_name 
-#  and uses a tag named test.
-#     <user_name>/metplus:test
-#
-# Configuration assumptions
-# These are configurable
-#  DOCKERHUB_USE_DEFAULT
-#  branches to build: develop
-#
- 
 language: python
-dist: xenial #Ubuntu Xenial 16.04
+dist: xenial
 os: linux
-
-# Travis docs indicate, need define the docker service, was able 
-# to run docker without this ... basically it runs this command
-# to start the service: sudo systemctl start docker
-# But note, the docker service is started and running anyway.
-# How do you know, If you comment out and run this command to show
-# the state of all services, is shows that docker is running.
-# systemctl list-units --type=service
 services:
-  - docker
-
-# Note: even though defining python versions with the 
-# python key which expands jobs...
-# We are not running METplus in these versions since
-# METplus is running in the docker container space
-# NOT on the travis vm space.
+- docker
 python:
-  - 3.6
-
+- 3.6
 git:
   depth: 25
-
-# This defines what is allowed to build.
-# Build only the safelist
 branches:
   only:
   - develop
-  - /^feature_.*/
-  - /^main_.*/
-  - /^bugfix_.*/  
-
+  - "/^feature_.*/"
+  - "/^main_.*/"
+  - "/^bugfix_.*/"
 cache:
-  - pip
-
-# HOME=/home/travis
-# TRAVIS_BUILD_DIR=/home/travis/build/owner_name/repo_name
-# TRAVIS_REPO_SLUG=owner_name/repo_name
-# OWNER_BUILD_DIR=/home/travis/build/owner_name
+- pip
 env:
   global:
-    - DO_GIT_CLONE=false
-    - REPO_NAME=METplus
-    - REPO_NAME_lc=metplus
-    - CURRENT_BRANCH=${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}
-    - GH_PAGES_DIR=${HOME}/gh-pages
-    - PYTHON_PACKAGES="requests"
-    - OWNER_BUILD_DIR=`dirname ${TRAVIS_BUILD_DIR}`
-    - DOCKER_WORK_DIR=/metplus
-    - GITHUB_REPO_ORG=DTCenter
-    - DOCKERHUB_DEFAULT_REPO=dtcenter/metplus
-    - DOCKERHUB_DEFAULT_TAGNAME=develop
-    - DOCKERHUB_MET_TAGNAME=develop
-    - DOCKER_DATA_DIR=/data
-    # The DOCKERHUB_USER_REPO setting ASSUMES your owner_name 
-    # in travis ci IS THE SAME as your docker hub user_name,
-    # since it is setting it  based on the travis repo slug.
-    - DOCKERHUB_USER_REPO="`dirname ${TRAVIS_REPO_SLUG}| awk '{print tolower($0)}'`/metplus"
-    - DOCKERHUB_USER_TAGNAME=develop
-    # set to true to use default settings
-    # if set to false, to use DOCKER_USER_* settings, your dockerhub container.
-    - DOCKERHUB_USE_DEFAULT=true
+  - DO_GIT_CLONE=false
+  - REPO_NAME=METplus-dev
+  - REPO_NAME_lc=metplus-dev
+  - CURRENT_BRANCH=${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}
+  - GH_PAGES_DIR=${HOME}/gh-pages
+  - PYTHON_PACKAGES="requests"
+  - OWNER_BUILD_DIR=`dirname ${TRAVIS_BUILD_DIR}`
+  - DOCKER_WORK_DIR=/metplus
+  - GITHUB_REPO_ORG=DTCenter
+  - DOCKERHUB_DEFAULT_REPO=dtcenter/metplus-dev
+ #- DOCKERHUB_DEFAULT_TAGNAME=develop
+  - DOCKERHUB_DEFAULT_TAGNAME=feature_639_dockerimagepushpull
```

Comment:
Setting this to the current branch of this PR will result in every branch using that DockerHub image. I am about to push some changes to resolve this.